src/js/components/rebus.js

```diff
@@ -11,7 +11,11 @@ export function Rebus(props, ...children) {
         this.$element.querySelector('input').focus();
       },
       componentDidUpdate() {
-        this.$element.querySelector('input').focus();
```

Comment:
As of #169 you can access the props of a connected component using `this.props`. This means that there is no need to use localStorage to save any state. To get the current rebus, you can just do something like:
```js
const rebus = this.props.rebuses[this.props.current];
```

If the rebus is not answered, apply focus to the first char input:
```js
this.$element.querySelector('input').focus();
```

Else, focus the next button:
```js
this.$parent.querySelector('.change-button--next').focus();
```

You will have to do this in both componentDidMount and componentDidUpdate (unfortunately) 🙂

Reply:
Thank you very much for your feedback 😊  I will do it asap!